packages/appcues-custom-previewer/appcues-custom-previewer.podspec

```diff
@@ -0,0 +1,20 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = "appcues-custom-previewer"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => "11.0" }
```

Comment:
side note, does showcase support down to 11? Maybe can just be 13 since most have that at least now, and it's really not going to showcase much < 13 😄

Reply:
The app as a whole is 13+